aztec/src/main/kotlin/org/wordpress/aztec/watchers/DeleteMediaElementWatcherPreAPI25.kt

```diff
@@ -10,17 +10,18 @@ class DeleteMediaElementWatcherPreAPI25(aztecText: AztecText) : TextWatcher {
     private val aztecTextRef: WeakReference<AztecText?> = WeakReference(aztecText)
 
     override fun beforeTextChanged(text: CharSequence, start: Int, count: Int, after: Int) {
-        if (aztecTextRef.get()?.isTextChangedListenerDisabled() ?: true) {
+        if (aztecTextRef.get()?.isTextChangedListenerDisabled() != false) {
```

Comment:
This changes the logic of the original if condition. Was this intentional? The equivalent to what is happening originally would be:

```
if (aztecTextRef.get() == null || aztecTextRef.get()?.isTextChangedListenerDisabled() != false) {
```

Reply:
This was offered by the Android Studio and I don't think it changes the logic. Both the conditions are true when the `aztecTextRef.get()?.isTextChangedListenerDisabled() == null` because `null != false` is true. The only case when both are false are if the `aztecTextRef != null && isTextChangedListenerDisabled == false`